qa/tasks/cephfs/test_damage.py

```diff
@@ -437,11 +436,7 @@ def test_damaged_dentry(self):
         try:
             self.mount_a.stat("subdir/file_to_be_damaged", wait=True)
         except CommandFailedError as e:
-            if isinstance(self.mount_a, FuseMount):
-                self.assertEqual(e.exitstatus, errno.EIO)
-            else:
-                # Kernel client handles this case differently
-                self.assertEqual(e.exitstatus, errno.ENOENT)
+            self.assertEqual(e.exitstatus, errno.EIO)
```

Comment:
I tried to double-check that this is a new behavior in the `testing` branch of the kernel client but all the tests from the ticket ran with the testing branch.  So we don't have a comparison.

I worry this is a new behavior and tests with the stock kernel will fail with this code change.

Reply:
Or let's check the both errnos ? If any of them matches will the test case  success ?

Reply:
Makes sense to me.